future-apps/actor/contracts/Actor.sol

```diff
@@ -17,7 +17,6 @@ contract Actor is Vault, IERC165, IERC1271, IForwarder {
     bytes32 public constant EXECUTE_ROLE = keccak256("EXECUTE_ROLE");
     bytes32 public constant RUN_SCRIPT_ROLE = keccak256("RUN_SCRIPT_ROLE");
     bytes32 public constant PRESIGN_HASH_ROLE = keccak256("PRESIGN_HASH_ROLE");
```

Comment:
Given this change, we need a better name for the role

Reply:
I can't really think of a better name. What do you think it should be? It's still presigning hashes.